Sources/gg.datagram.web-requests.sdPlugin/app.js

```diff
@@ -18,8 +18,34 @@ function connected(jsn) {
  * }} data
  */
 function sendHttp(data) {
-    const { url, method, body } = data.payload.settings;
-    log('sendHttp', { url, method, body });
+    const { url, method, contentType, headers, body } = data.payload.settings;
+    log('sendHttp', { url, method, contentType, headers, body });
+
+    let defaultHeaders = {
+      'Content-Type': contentType
```

Comment:
`contentType` can be `undefined`; we need to handle this, otherwise we'll end up sending a header with a value of `"undefined"`.